python/src/opendp/context.py

```diff
@@ -413,38 +416,47 @@ class Context(object):
     :param d_mids: A sequence of privacy losses for each query to be sent to the queryable. Used for compositors.
     :param d_out: An upper bound on the overall privacy loss. Used for filters."""
 
-    accountant: Measurement  # union Odometer once merged
+    accountant: Union[Measurement, Odometer]
     """The accountant is the measurement used to spawn the queryable.
     It contains information about the queryable, 
     such as the input domain, input metric, and output measure expected of measurement queries sent to the queryable."""
-    queryable: Queryable
+    queryable: Union[Queryable, OdometerQueryable]
     """The queryable executes the queries and tracks the privacy expenditure."""
 
     def __init__(
         self,
         accountant: Measurement,
         queryable: Queryable,
         d_in: Union[float, Sequence[Bound]],
-        d_mids: Optional[Sequence[float]] = None,
-        d_out: Optional[float] = None,
-        space_override: Optional[
-            tuple[Domain, Metric]
-        ] = None,  # TODO: Document or add leading underscore and explain that is is for internal use only.
+        d_mids: Optional[Sequence[Union[float, tuple[float, float]]]] = None,
+        d_out: Optional[Union[float, tuple[float, float]]] = None,
+        _space_override: Optional[tuple[Domain, Metric]] = None,
```

Comment:
Still shows up in API docs: Maybe people should infer that it is internal from the underscore, but a docstring might still be good, for developers if not for end users.